src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -290,6 +293,7 @@ Command StdIO()
                 command.Handler = CommandHandler.Create<StartupOptions, StdIOOptions, IConsole, InvocationContext>(
                     (startupOptions, options, console, context) =>
                     {
+                        ExtendProtocol();
```

Comment:
Can we give this a name that explains the why rather than the what?

Reply:
```UseQuitCommand()``` ?

Reply:
Done